docker/debian-base.dockerfile

```diff
@@ -1,12 +1,11 @@
-# DON'T UPDATE TO node:14-bullseye-slim, see #372.
 # If the image changed, the second stage image should be changed too
-FROM node:16-buster-slim
+FROM node:20-bookworm-slim AS base2-slim
```

Comment:
Omg, I am just curious about why the Debian version - Bookworm sounds a bit wired. Then I just realized that all version names are actually Toy Story character names. I didn't watch the English version of Toy Story, I never know their names though, except Woody and Buzz.